wiki/People/The_Team/Account_support_team/zh-tw.md

```diff
@@ -0,0 +1,45 @@
+# 帳號支援團隊
+
+*關於管理論壇的團隊，請見：[支援團隊](/wiki/People/The_Team/Support_Team)*
+
+**帳號支援團隊** 是管理玩家帳號的小組，同時也接受禁言及限制申訴。
+
+## 聯絡方式
+
+團隊**只能**透過以下三個電子郵件地址聯繫：[support@ppy.sh](mailto:support@ppy.sh)、[accounts@ppy.sh](mailto:accounts@ppy.sh)、或 [privacy@ppy.sh](mailto:privacy@ppy.sh)。團隊成員的身分為保密，以避免不必要的關注。
+
+團隊通常需要幾天才能處理你的信件。如果一周後你還沒收到回覆，可以再傳一封。
+
+## 角色與職責
+
+*主頁面：[幫助中心](/wiki/Help_Centre)*
+
+*另見：[帳號限制](/wiki/Help_Centre/Account_Restrictions)*
+
+當你遇到你無法控制的帳號問題時，請聯繫本團隊。記得使用你的 osu! 帳號綁定的電子郵件信箱發送信件，並盡可能地提供關於你的問題的詳細資訊。
+
+### [accounts@ppy.sh](mailto:accounts@ppy.sh)
+
+- [帳號限制申訴](/wiki/Help_Centre/Account_Restrictions), 包括：
+  - 禁言；
+  - 個人檔案內容遭移除；
+  - 圖譜遭移除；
+  - 論壇帖子或留言遭移除。
+- 你想[承認](/wiki/Reporting_Bad_Behaviour/Handling_Foul_Play#what-can-i-do-if-i've-broken-the-rules?)的不良行為。
+- [無法存取](/wiki/Help_Centre#sign-in)你的 osu! 帳號綁定的電子郵件信箱，或帳號被盜用。
+- [復原使用者名稱或修正拼寫錯誤](/wiki/Help_Centre#name-changes)。
+- [機器人帳號註冊](/wiki/Bot_Account)。
+
+### [support@ppy.sh](mailto:support@ppy.sh)
+
+*提醒：若要檢舉任何玩家 (包括管理員) 的極端不當行為或性騷擾，請見[檢舉性騷擾](/wiki/Reporting_Bad_Behaviour/Abuse)。*
+
+- 對於管理員不當行為的疑慮。
+- 對於管理員觸犯了[貢獻者行為準則](/wiki/Contributor_Code_of_Conduct)的疑慮。
+- [osu!store](https://osu.ppy.sh/store/listing) 的[付款問題](/wiki/en/Help_Centre#supporter)。
+- 無法或尚未在[幫助論壇](https://osu.ppy.sh/community/forums/5)獲得解決的技術性問題，例如[連線異常](/wiki/Help_Centre#online-features)。
+
+### [privacy@ppy.sh](mailto:privacy@ppy.sh)
+
+- [資料請求](/wiki/Legal/Privacy#data-controller)及隱私疑慮。
+- **永久**[帳號刪除請求](/wiki/Legal/Privacy#your-rights-and-control) (僅限在 [GDPR](https://zh.wikipedia.org/wiki/%E6%AD%90%E7%9B%9F%E4%B8%80%E8%88%AC%E8%B3%87%E6%96%99%E4%BF%9D%E8%AD%B7%E8%A6%8F%E7%AF%84 "維基百科") 管轄範圍內的歐盟居民可申請)。
```

Comment:
the GDPR note was removed, please apply 234321c76b4fe27bcd7d21524603a9d975701ecb to your translation